spec/features/user_can_see_a_single_post_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Single post link', type: :feature do
+  scenario 'Can see single post link' do
```

Comment:
I was confused about what this test was supposed to be doing.
Based on the name of the file, feature and scenario I was expecting a test for a link to a single post but the link is for all posts.
Maybe think about renaming things to make the intent a bit clearer?